scripts/time_plot.py

```diff
@@ -1840,18 +1840,18 @@ def handle_websock_event(handlerkey,*args):
                         obs_params_key=telstate.join(cbid, 'obs_params')
                         obs_params=telstate.get(obs_params_key, {})
                         for obskey,obsvalue in obs_params.items():
-                            send_websock_cmd('logconsole("'+obskey+': '+repr(obsvalue)+'",true,true,true)',handlerkey)
+                            send_websock_cmd('logconsole("'+repr(obskey)+': '+repr(obsvalue)+'",true,true,true)',handlerkey)
                     elif (thekey in telstate):
                         if telstate.is_immutable(thekey):
-                            send_websock_cmd('logconsole("'+thekey+': '+repr(telstate[thekey])+' (immutable, not plottable)",true,true,true)',handlerkey)
+                            send_websock_cmd('logconsole("'+repr(thekey)+': '+repr(telstate[thekey])+' (immutable, not plottable)",true,true,true)',handlerkey)
```

Comment:
Why? `thekey` is already a string, so now you're going to get an extra layer of quoting around it.

Reply:
thekey is not always a string

Reply:
maybe in this case it might be a string but there are other examples of this where it wasnt and caused an error where it turned out to be a byte string.

Reply:
How can it not be a string when it's defined as `thekey=str(args[1])` at line 1837?